src/Aggregation/ExtendedStatsBucket.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Elastica\Aggregation;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Implements a Extended Stats Bucket Aggregation.
+ *
+ * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-extended-stats-bucket-aggregation.html
+ */
+class ExtendedStatsBucket extends AbstractAggregation
+{
+    public function __construct(string $name, ?string $bucketsPath = null)
```

Comment:
I just noticed it now: why is the bucketPath not required in the constructor, but later required when building the request( `toArray()`)?

As it is required, I'd make it required (not empty) and remove the `setBucketPath()` method. What do you think?

Reply:
@nicolassing ^^

Reply:
@thePanz Perhaps the same argument as in the other PR might apply. No strong opinion which on is better but we should become consistent.

Reply:
i followed the model of the StatsBucket aggregation :/

But both of you are right. We can make the $bucketsPath argument mandatory and remove the setter in the 2 classes :)

Reply:
@nicolassing : I'd leave the setters (and do the "not empty" check there, throwing an exception)
and make the argument required in the constructor

I'll cleanup the other aggregations @ruflin , but in another PR. I'd like not to have devs creating invalid objects to start with :) but this is a topic for a discussion, not here tho :)

Reply:
SGTM, lets try it out.

Reply:
But first: let's have the `$bucketsPath` argument as required :)